buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersionHelper.kt

```diff
@@ -54,4 +54,18 @@ object IntellijVersionHelper {
   private fun getFile(): File {
     return File("intellijVersions.properties")
   }
+
+  infix fun String.versionIsNewerThan(rhsBuildNumber: String): Boolean {
+    val lhsSplit = this.split('.')
+    val rhsSplit = rhsBuildNumber.split('.')
```

Comment:
Hmmm `buildNumber`? 🤔 I wouldn't suggest that a `222.1234.56` build number is expected here... pls smooth out naming